### PR TITLE
Create and Edit Product Property with UI components issue #5857

### PR DESCRIPTION
**Pull request by Astr0surf3r**
## Summary

fix the issue [#5857](https://github.com/solidusio/solidus/issues/5857) 

Now to add a product property appear a modal

window-new-property

Screenshot 2024-10-30 at 5 26 26 p m

And to edit a Property appear a modal as well

window-edit-property

success-updated

[x] I have written a thorough PR description.
[x] I have kept my commits small and atomic.
[x] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

✅ I have added automated tests to cover my changes.
📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: admin/app/components/solidus_admin/properties/edit/component.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+class SolidusAdmin::Properties::Edit::Component < SolidusAdmin::BaseComponent
+    def initialize(page:, property:)
+      @page = page
+      @property = property
+    end
+
+    def form_id
+      dom_id(@property, "#{stimulus_id}_edit_property_form")
+    end
+  end
```
Comment (jarednorman) [inline admin/app/components/solidus_admin/properties/edit/component.rb:12]: This file is mostly misindented.

**Review by jarednorman — CHANGES_REQUESTED** [review]
@Astr0surf3r You have not followed [our pull request guidelines](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#pull-request-guidelines):

- The commit messages lack any explanation.
- Some commits add unnecessary code that gets removed by later commits.
- Linting changes shouldn't be in separate commits and your linting commit also removes unneeded code.

I'd appreciate a rebase of this PR to address these issues.

@MadelineCollier Can you take a pass over this from an implementation perspective and make sure it's consistent with how similar features are built? You have better knowledge of how admin features should be built.

File: admin/app/components/solidus_admin/properties/edit/component.rb (hunk 1 above)
Comment (Astr0surf3r) [reply admin/app/components/solidus_admin/properties/edit/component.rb:11]: @jarednorman thank you for your message I have just already made rebase in this PR and follow the solidus guideline let me know if need a cleaner PR I can upload 1 single commit with all changes ... let me know please

**Comment by tvdeyen** [comment]
@Astr0surf3r Please rebase this branch wit latest `main` and make one commit with your changes only. Thanks for the contribution.

**Comment by jarednorman** [comment]
I think this looks reasonable, but can you please rebase this PR against our main branch. I'm not sure what went wrong with your previous rebase, but you've ended up with a bunch of extra commits in your PR that shouldn't be there.

**Comment by MadelineCollier** [comment]
The work started here has been pulled into https://github.com/solidusio/solidus/pull/6011. Thanks for your contribution @Astr0surf3r !

**Pull request closed without merging**